addons/website/static/src/snippets/s_company_team/000.scss

```diff
@@ -5,4 +5,11 @@
             max-width: 50%;
         }
     }
+    .container-fluid {
+        .o_grid_mode {
+            .row:last-child {
+                margin-right: 0;
```

Comment:
@lucasth-dev That's not the right fix, for multiple reasons:

- First given the nature of the bug, this is really not something that is only present for the company team snippet but a general issue of the grid mode: any row as a direct child of a grid item would have the problem. The fix should thus not be here but in the grid mode general CSS.

- "Any row as a direct child of a grid item has the problem" => Indeed the issue is that we do not enforce that the padding of a grid item (which the user can control) is bigger than the gutter-grid i.e., the space that rows compensate (and sub-columns uses as padding). => I think the fix is indeed changing the negative margin of those rows but not to 0: setting it to compensate the exact size of the grid item padding.

cc @sobo-odoo You might be interested to know :)